EmarsysInbox/EmarsysInboxConfig.swift

```diff
@@ -8,6 +8,7 @@ public class EmarsysInboxConfig {
     
     public static var headerBackgroundColor: UIColor? = .color(fromHexCode: 0x5F9F9FFF)
     public static var headerForegroundColor: UIColor? = .white
+    public static var headerView: UIView = EmarsysInboxDefaultHeaderView(backgroudColor: headerBackgroundColor, titleColor: headerForegroundColor)
```

Comment:
it is not making use of the custom `headerBackgroundColor` / `headerForegroundColor` that will be configured in the clients project during init.
i suggest the following:
```
public static var headerView: UIView?
```
and in `EmarsysInboxController` :
```
headerView.addSubview(EmarsysInboxConfig.headerView ?? EmarsysInboxDefaultHeaderView(backgroudColor: EmarsysInboxConfig.headerBackgroundColor, titleColor: EmarsysInboxConfig.headerForegroundColor))
```
what do you think?